From 43de7f5e9c900cf9e2464d2e8bdd9bce12c5b576 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 27 Jul 2026 16:22:08 +0100
Subject: [PATCH] overlays: apds9960: Add missing pinctrl reference

The apds9960 overlays declares its pin usage (one input for interrupts),
but doesn't include a reference from the driver node. This seems to be
harmless - declaring inputs and outputs for GPIOs is almost certainly
unnecessary - but it is flagged as an error by an improved version of
the overlaycheck utility.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/apds9960-overlay.dts | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/boot/dts/overlays/apds9960-overlay.dts
+++ b/arch/arm/boot/dts/overlays/apds9960-overlay.dts
@@ -41,6 +41,8 @@
 			apds9960: apds@39 {
 				compatible = "avago,apds9960";
 				reg = <0x39>;
+				pinctrl-names = "default";
+				pinctrl-0 = <&apds9960_pins>;
 				status = "okay";
 			};
 		};
